### ISSUE TITLE
Feature: Group Team-U people by Nano Degree

### ISSUE THREAD

**Issue by scrabill** [Team-U/Team-U.github.io#3]
I was thinking for when this list gets longer, it may be helpful to have people grouped by some common factor. At first I was thinking current title, but that may not be relevant since some people may be changing jobs or have skills that do not reflect their title. 

How about grouping people by the nanodegree (or degrees) that they have completed? Something like this

### Mobile Web Specialist
* **Guy Mann** - [Github](https://github.com)

OR

Include the NanoDegree as part of their entry

## Who we are:

* **Jane Doe** - Android Basics, 2018 - [Github](https://github.com)

OR, another option entirely, encourage people to include a list of a few interests along with their contact info

* **Dana Scully** - React, Python, Open Source - [Github](https://github.com)

**Comment by chadpjontek** [comment]
Yeah, I'm definitely up to changes on this. Jerome said he wanted to work on a landing page for us. Whatever we decide I would like to have a good structure and visibility for new visitors to see us as well as make it easy for newcomers to our group by being able to a simple PR.

**Comment by patriciarrsilva** [comment]
I like the first suggestion better (grouping by nanodegree) :)

**Comment by chadpjontek** [comment]
@PlaySnowi I do like how it looks with each Nanodegree section but there are some people that have completed multiple ones. Perhaps @scrabill idea of including it as part of the entry?

**Comment by coding-catie** [comment]
Having the nanodegree(s) as part of the entry sounds good. We can just make sure that upon merging their entry, that it is properly capitalized and such (unless that doesn't bother anyone else, lol).

**Comment by chadpjontek** [comment]
That sounds good @catielynncodes . I think it will look better if it's uniform so yeah we should capitalize them. We can also add links to the certs themselves so like:
* **Chad Pjontek** - [Mobile Web Specialist](https://confirm.udacity.com/PQPKPWKC) / [Front End Developer](https://confirm.udacity.com/PQPKPWKC) - [Github](https://github.com/chadpjontek) / [LinkedIn](https://www.linkedin.com/in/chad-pjontek/)

Does everyone think that is ok?

P.S. I'm not actually FE certified I just put as an example!

**Comment by ATMalcolm** [comment]
Looks good to me.

**Comment by DavidJBrin** [comment]
Organizing by program helps with team building too so that way as we expand and start relying on subject matter expertise and refining bits of projects, we can target based on initial program track.

**Comment by chadpjontek** [comment]
Ok I just changed the instructions to include Nanodegrees in the entry for those that have obtained them.

**Comment by notacouch** [comment]
I think it helps to include what role we'll play in our careers and/or here. E.g. my nanodegree is MWS, so I like the convention Chad started with linking to that (grad cert if you completed, or scholarship cert if that's the last thing you got from Udacity). But that doesn't say anything about my frontend/backend/fullstack chops (nor should someone have to peruse through our github, linkedin, etc. to figure that out). More importantly, I would prefer to minimize non-frontend activity. So by saying somewhere, "Frontend Developer", it helps indicate for others where this member might fit or want to fit. And since it wouldn't be a link, it'll already have contrast from the other stuff next to our names. 

We could always make this aspect a wiki entry instead of part of the README, no?

**Comment by chadpjontek** [comment]
I agree that we need a way of sharing more about ourselves. This README is a first step of getting people involved quickly and easily with our group. I set this repo up to be a Github pages site so we can create a nice front-end experience for everyone. When it is finished we can link to it from here.

 I've just created a team called **Team-U.github.io Contributors**. This team will be responsible for developing and maintaining our main Github Pages site. Drop a line here or in our [new Slack](https://join.slack.com/t/team-u-devs/shared_invite/enQtNDYwNzU3NjI4NzU0LTUwZmMzYjQ1ODIzYTI0NjE4N2Y2YTRkYmEzNTg5ZWVmODVmMTJhYmZiNjU1Y2M0ZmEyZGRiOWYwYjlhNjU1ZDY) if you are interested in contributing to this!

I want to keep this discussion open for a while because I'd like as much feedback as possible but I'm also going to be making new team specific issues for getting the ball rolling on the new site.

**Comment by notacouch** [comment]
(complete and total tangent:
![image](https://user-images.githubusercontent.com/2334143/47382029-57897280-d6cf-11e8-88b3-8ac29691bff3.png)

GitHub needs animated GIF support right there lol)
